docs/en/wallet-instance-attestation.rst

```diff
@@ -2,56 +2,676 @@
+
+Verifiable through the public key of the wallet instance present in the JWT.
+
+
+Format of the Wallet Instance Attestation
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To ensure the above requirements, a JWT was chosen as the format for the
+Wallet Instance Attestation. Let's see below the various fields that
+compose it
+
+Header
+^^^^^^
+
++-----------------------------------+-----------------------------------+
+| **key**                           | **value**                         |
++-----------------------------------+-----------------------------------+
+| alg                               | Algorithm to verify the token     |
+|                                   | signature (es. ES256)             |
++-----------------------------------+-----------------------------------+
+| kid                               | Key id used by the wallet         |
+|                                   | provider to sign the attestation  |
++-----------------------------------+-----------------------------------+
+| type                              | Media type, in this case we use   |
+|                                   | the value va+jwt (Verifiable      |
+|                                   | Assertion JWT).                   |
+|                                   | This parameter is currently       |
+|                                   | non-standard as it is not yet     |
+|                                   | registered as `IANA Media         |
+|                                   | Types <https://www.iana.org/assig |
+|                                   | nments/media-types/media-types.xh |
+|                                   | tml>`__                           |
++-----------------------------------+-----------------------------------+
+| x5c                               | Array containing the X.509        |
+|                                   | certificate (and the entire chain |
+|                                   | of certificates) used to certify  |
+|                                   | the public key of the issuer.     |
++-----------------------------------+-----------------------------------+
+| trust_chain                       | | Array containing the JWS of the |
+|                                   |   trust chain relating to its     |
+|                                   |   issuer (Wallet Provider).       |
+|                                   | | ⚠️ This topic will be addressed |
```

Comment:
trust_chain will be addresses with the first implementation, aside x5c, since x5c will be published in federation entity statements

Reply:
so advice to delete from here only x5c ?

Reply:
no please keep it just for now :-)